org.mume.MMapper.json

```diff
@@ -0,0 +1,167 @@
+{
+  "app-id": "org.mume.MMapper",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.8",
+  "sdk": "org.kde.Sdk",
+  "finish-args": [
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=xdg-documents:rw"
+  ],
+  "command": "mmapper",
+  "cleanup": [
+    "/app/images",
+    "/app/include",
+    "/app/arda.xml",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "openssl3",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/openssl/openssl.git",
+          "tag": "openssl-3.5.1"
+        }
+      ],
+      "build-commands": [
+        "./config shared no-tests no-unit-test no-weak-ssl-ciphers no-idea no-mdc2 no-rc5 --prefix=/app",
+        "make",
+        "make install"
+      ]
+    },
+    {
+      "name": "glm",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/g-truc/glm.git",
+          "tag": "0.9.9.7"
+        }
+      ],
+      "build-commands": [
+        "cp -r glm /app/include/"
+      ]
+    },
+    {
+      "name": "immer",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/arximboldi/immer.git",
+          "tag": "v0.8.1"
+        }
+      ],
+      "build-commands": [
+        "cp -r immer /app/include/"
+      ]
+    },
+    {
+      "name": "libsecret",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dmanpage=false",
+        "-Dvapi=false",
+        "-Dgtk_doc=false",
+        "-Dintrospection=false"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz",
+          "sha256": "163d08d783be6d4ab9a979ceb5a4fecbc1d9660d3c34168c581301cd53912b20",
+          "x-checker-data": {
+            "type": "gnome",
+            "name": "libsecret",
+            "stable-only": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "qtkeychain",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/frankosterfeld/qtkeychain/archive/0.14.3.tar.gz",
+          "sha256": "a22c708f351431d8736a0ac5c562414f2b7bb919a6292cbca1ff7ac0849cb0a7",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 4138,
+            "stable-only": true,
+            "url-template": "https://github.com/frankosterfeld/qtkeychain/archive/$version.tar.gz"
+          }
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+        "-DLIB_INSTALL_DIR=/app/lib",
+        "-DBUILD_TRANSLATIONS=OFF",
+        "-DBUILD_WITH_QT6=ON",
+        "-DCMAKE_PREFIX_PATH=/app"
```

Comment:
```suggestion
        "-DBUILD_WITH_QT6=ON"
```

not needed.

Reply:
It required `"-DBUILD_WITH_QT6=ON"` or it would attempt to build with Qt5

Reply:
Correct, I misread. I've corrected this now!